core/src/test/java/io/cucumber/core/model/FeaturePathTest.java

```diff
@@ -82,12 +82,31 @@ public void can_parse_windows_absolute_path_form(){
         assertEquals(file.toURI().getSchemeSpecificPart(), uri.getSchemeSpecificPart());
     }
 
-
     @Test
     public void can_parse_whitespace_in_path(){
         URI uri = FeaturePath.parse("path/to the/file.feature");
         assertEquals("file", uri.getScheme());
         assertEquals("path/to the/file.feature", uri.getSchemeSpecificPart());
     }
+    
+    @Test
+    public void can_parse_windows_file_path_with_standard_file_separator(){
+        assumeThat(System.getProperty("os.name"), containsString("Windows")); //Requires windows
+        URI uri = FeaturePath.parse("C:/path/to/the/file.feature");
+        assertEquals("file", uri.getScheme());
+        // Use File to work out the drive letter
+        File file = new File("/path/to/the/file.feature");
+        assertEquals(file.toURI().getSchemeSpecificPart(), uri.getSchemeSpecificPart());
+    }
+
+    @Test
+    public void can_parse_windows_file_path_with_standard_file_separator_and_non_alpanumeric_chars(){
+        assumeThat(System.getProperty("os.name"), containsString("Windows")); //Requires windows
+        URI uri = FeaturePath.parse("C:/path-to/the_file.feature");
+        assertEquals("file", uri.getScheme());
+        // Use File to work out the drive letter
+        File file = new File("/path-to/the_file.feature");
```

Comment:
I now realize that the test this was copied from declares the current drive letter to be `C`. So File is not needed here, nor in `can_parse_windows_file_path_with_standard_file_separator`, nor in `can_parse_windows_absolute_path_form`.

It would have been useful in a relative path test on Windows but we don't seem to have a test case for that.

Reply:
Good point - the aim of the test was to make sure that the scheme was being reolved as file, no tthe drive letter.  Updated test accordingly

Reply:
True. But it's good to test the path anyway. The drive letter shouldn't influence it. It should always be `C` because thats what is used in the setup.